index.html

```diff
@@ -0,0 +1,191 @@
+                           <span></span>
+                       </label>
+                   </div>
+                   <div class="fld-mapviz-filter fld-mapviz-filter-washrooms">
+                       <span class="fld-mapviz-filter-icon"></span>
+                       <span class="fld-mapviz-filter-tooltip-icon flc-mapviz-hoverable"></span>
+                       <span class="fld-mapviz-filter-tooltip"></span>
+                       <label class="fld-mapviz-filter-title" for="toggle-2">Accessible Washrooms</label>
+                       <label class="flc-checkbox-holder">
+                           <input class="fld-mapviz-checkbox visually-hidden" id="toggle-2" type="checkbox"/>
+                           <span></span>
+                       </label>
+                   </div>
+                   <div class="fld-mapviz-filter fld-mapviz-filter-parking">
+                       <span class="fld-mapviz-filter-icon"></span>
+                       <span class="fld-mapviz-filter-tooltip-icon flc-mapviz-hoverable"></span>
+                       <span class="fld-mapviz-filter-tooltip"></span>
+                       <label class="fld-mapviz-filter-title" for="toggle-3">Accessible Parking</label>
+                       <label class="flc-checkbox-holder">
+                           <input class="fld-mapviz-checkbox visually-hidden" id="toggle-3" type="checkbox"/>
+                           <span></span>
+                       </label>
+                   </div>
+                   <div class="fld-mapviz-filter fld-mapviz-filter-individual">
+                       <span class="fld-mapviz-filter-icon"></span>
+                       <span class="fld-mapviz-filter-tooltip-icon flc-mapviz-hoverable"></span>
+                       <span class="fld-mapviz-filter-tooltip"></span>
+                       <label class="fld-mapviz-filter-title" for="toggle-4">Individual Service</label>
+                       <label class="flc-checkbox-holder">
+                           <input class="fld-mapviz-checkbox visually-hidden" id="toggle-4" type="checkbox"/>
+                           <span></span>
+                       </label>
+                   </div>
+                   <div class="fld-mapviz-filter fld-mapviz-filter-wait">
+                       <span class="fld-mapviz-filter-icon"></span>
+                       <span class="fld-mapviz-filter-tooltip-icon flc-mapviz-hoverable"></span>
+                       <span class="fld-mapviz-filter-tooltip"></span>
+                       <label class="fld-mapviz-filter-title" for="toggle-5">Wait Accommodations</label>
+                       <label class="flc-checkbox-holder">
+                           <input class="fld-mapviz-checkbox visually-hidden" id="toggle-6" type="checkbox"/>
+                           <span></span>
+                       </label>
+                   </div>
+                </div>
+                <div class="fld-mapviz-filter-buttons fl-mapviz-even-row">
```

Comment:
Is there a reason that css classes for this element and elements at line 123 & 127 named `fl-mapviz-even-row` for these buttons?

Reply:
Yes, this ensures that the button labels align with their images in the row: https://github.com/inclusive-design/covid-data-monitor/blob/d33e874ab56b659cbf1e2c5e26fd48ba8e38b312/src/css/fluid-covid-map-viz.css#L181

Reply:
Got it. Thanks for the explanation.